projects/cardiovascular-patient-app/src/app/report-bp/report-bp.component.ts

```diff
@@ -19,6 +19,8 @@ export class ReportBPComponent implements OnInit {
   submitted = false;
   min = 11;
   max = 250;
+  
+  @Output() selectedIndex = new EventEmitter();
```

Comment:
Please rename this output to something like "bpSubmitted" or "resourceCreated". This component does not have any concept of selected index.